README.md

```diff
@@ -1,8 +1,7 @@
-# <!-- H1 title omitted because our logo acts as the title. -->
-
+<!-- H1 title omitted because our logo acts as the title. -->
 <div align="center">
-<img width="280px" height="135px" alt="Cirq logo"
-src="https://raw.githubusercontent.com/quantumlib/Cirq/refs/heads/main/docs/images/Cirq_logo_color.svg">
+<img alt="Cirq logo" width="340px" src="docs/images/Cirq_logo_black.svg#gh-light-mode-only">
+<img alt="Cirq logo" width="340px" src="docs/images/Cirq_logo_white.svg#gh-dark-mode-only">
```

Comment:
We need an absolute URL here otherwise the logo will not show at https://pypi.org/project/cirq/1.5.0.dev20250325195502/.

Is this going to render well at PyPI?

Reply:
> We need an absolute URL here otherwise the logo will not show at [pypi.org/project/cirq/1.5.0.dev20250325195502](https://pypi.org/project/cirq/1.5.0.dev20250325195502/).

I had a chicken and egg problem: can't provide an absolute URL until the SVG files are added to the repo.

Reply:
> Is this going to render well at PyPI?

Good thing you asked. I had previously done this sort of thing for past projects, but didn't use the approach from before (which involves `<picture>` elements) because I forgot which I had it that way in the first place. (Sigh.) I added it to the latest commit.

This now does work on PyPI. PyPI doesn't have dark mode support, so all one ever gets there is the light mode version. The scheme here has a default for that reason.

Reply:
It seems possible to adjust the SVG file instead so it picks different colors for light and dark mode - https://stackoverflow.com/q/67187091.

setuptools appears to use it in their favicon here - 
https://github.com/pypa/setuptools/blob/b545fc778583f644d6c331773dbe0ea53bfa41af/docs/images/favicon.svg?short_path=a1d3191#L3-L10 and it indeed renders differently in 

light mode
![image](https://github.com/user-attachments/assets/e19d4eb4-5c19-48f7-b574-6164fdd527be)

and dark mode
![image](https://github.com/user-attachments/assets/db11e6cf-034c-4a64-8eff-92fc95c4fba9)

Reply:
I had actually seen that S.O. posting but didn't want to spend figuring out and debugging the solution. However, I ended up doing that for Qualtran, and it turned out not to be as bad as I feared.

You're right that this is the best solution. Next push will have the changes to use this approach.